Source/WebCore/platform/graphics/Font.h

```diff
@@ -78,11 +84,11 @@ DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(Font);
 class Font : public RefCounted<Font>, public CanMakeWeakPtr<Font>, public CanMakeCheckedPtr {
     WTF_MAKE_FAST_ALLOCATED_WITH_HEAP_IDENTIFIER(Font);
 public:
-    // Used to create platform fonts.
-    enum class Origin : bool { Remote, Local };
-    enum class Interstitial : bool { No, Yes };
-    enum class Visibility : bool { Visible, Invisible };
-    enum class OrientationFallback : bool { No, Yes };
+    using Origin = FontOrigin;
+    using Interstitial = FontInterstitial;
```

Comment:
Not your fault but since we're changing this. The name isn't good. This should be named `IsInterstitial / FontIsInterstitial`.

Reply:
I agree, I actually thought about that. I'll change it.